Packages/com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -116,6 +120,10 @@ internal class PostProcessPass : ScriptableRenderPass
 
         Material m_BlitMaterial;
 
+        private bool resolveDepth;
```

Comment:
Like we discussed earlier, we should already have the info on whether or not depth resolve is needed inside URP. You probably can reuse the same code to determine if we need to resolve depth or not.